tests/scripts/utils.sh

```diff
@@ -100,15 +85,10 @@ function run_suite() {
   
   install_ols "$1" "$3" "$4" "$5" "$6" "$7" "$8" "$9"
 
-  # wait for the ols api server to come up
-  wait_for_ols "$OLS_URL"
-  if [ $? -ne 0 ]; then
-    echo "Timed out waiting for OLS to become available"
-    echo "${JUNIT_TEMPLATE}" | sed "s/SUITE_ID/${1}/g" > $ARTIFACT_DIR/junit_setup_${1}.xml
```

Comment:
JUNIT_TEMPLATE(actually apparently it was JUNIT_SETUP_TEMPLATE) isn't used anymore, so you can delete it.

But fundamentally also the junit failure that JUNIT_TEMPLATE was used to generate, needs to be replicated by your new logic (i.e. we should still see a junit test failure representing the fact that OLS didn't become ready)

ironically also i think there was a bug here because it should have been JUNIT_SETUP_TEMPLATE, not JUNIT_TEMPLATE.  In any case the template is not needed now because replacing that template/generated junit test result is part of what your work is intended to replace.